### ISSUE TITLE
Enable fixed audio volume for all users checkbox is missing on Android

### ISSUE THREAD

**Issue by kirill-jjj** [BearWare/TeamTalk5#2224]
### Description

On android when creating a channel there is no checkbox to Enable fixed audio volume for all users, but when a channel is created, this setting is enabled by default, and cannot be turned off without a computer.

### Application

- [ ] qtTeamTalk
- [X] TeamTalkAndroid
- [ ] iTeamTalk
- [ ] TeamTalkClassic
- [ ] TeamTalkServer

### Platform

- [ ] Windows
- [ ] macOS
- [X] Android
- [ ] iOS
- [ ] Linux

### Expected behavior

The enable fixed volume for all users checkbox should be present, and the default value of the setting should be consistent across platforms.

### Actual behavior

This checkbox is not available on Android.

### Steps to reproduce problem

Open channel creation dialog box, and you will see that the checkbox is not there.

**Comment by peter1384** [comment]
yeah this is very annoying, when this option is enabled people cannot talk with their normal microphones, also, android users cannot create stereo channels, the stereo option exists but setting it and creating the channel it doesn't sends that to the server. I want to fix it but i don't know where is the code which sends channel creation packates to the server

**Comment by kirill-jjj** [comment]
@poretsky maybe you know how to fix that?

**Comment by kirill-jjj** [comment]
what about this issue? if you don't have time to add a checkbox, at least change the default setting to disabled, like it is on a qt client.
